scss/_reboot.scss

```diff
@@ -96,13 +96,13 @@ body {
   @include focus-visible(); // 1
 }
 
-.js-focus-visible :focus:not([data-focus-visible-added]):not(.focus-ring),
-.js-focus-visible .focus:not([data-focus-visible-added]):not(.focus-ring) { // 2
+.js-focus-visible :focus:not([data-focus-visible-added]):not(.focus-ring):not(.form-select:invalid),
```

Comment:
Hmm forgot about this during the review but might need an update of the `// 2.` comment

Reply:
Just pushed a new version of the comment, could you please have a look?

Reply:
Sound great, thanks!